.tekton/pipeline/upgrade-tests.yaml

```diff
@@ -137,6 +137,101 @@ spec:
           workspace: upgrade-shared-workspace
         - name: source
           workspace: source
+    - name: fetch-repository-post-upgrade
+      runAfter:
+        - "tests-post-upgrade"
+      taskRef:
+        name: git-clone
+      workspaces:
+        - name: output
+          workspace: source
+      params:
+        - name: url
+          value: $(params.repo_url)
+        - name: revision
+          value: $(params.target_branch)
+    - name: plnsvc-downgrade
+      runAfter:
+        - "fetch-repository-post-upgrade"
+      workspaces:
+        - name: kubeconfig-dir
+          workspace: upgrade-shared-workspace
+        - name: source
+          workspace: source
+      params:
+        - name: repo_url
+          value: $(params.repo_url)
+        - name: revision
+          value: $(params.revision)
+        - name: target_branch
+          value: $(params.target_branch)
+      taskSpec:
+        workspaces:
```

Comment:
I think we already have such task defined in the [task](https://github.com/openshift-pipelines/pipeline-service/tree/main/.tekton/tasks) folder. Can we use the existing one and create a new task if not defined already. So that our pipeline looks clean